### ISSUE TITLE
Copy License XML files to the data repo

### ISSUE THREAD

**Issue by goneall** [spdx/license-list-XML#1523]
Once the legal team officially declares the XML files as a stable standard format for use outside the legal team, we should copy the XML source files to the [license-list-data](https://github.com/spdx/license-list-data) repository under a folder `license-xml`.

Originally, I was thinking of enhancing the license-list-publisher, but since these files are not transformed in any way, we can just add a line in the makefile to copy the file.  The line would be added immediately before this line: https://github.com/spdx/license-list-XML/blob/65dd995755eba6076e895e020a7c7f3a1bcbe06a/Makefile#L40

**Comment by jlovejoy** [comment]
@goneall - do we need to discuss any aspects of what it means to make the XML files as a stable standard format (other than saying that)?

**Comment by zvr** [comment]
@goneall your [last comment](https://github.com/spdx/spdx-spec/issues/46#issuecomment-919610984) on the issue you refer to about documenting the fields mentions

> I'm only suggesting we document the fields that are published on the website and in the license-list-data repo. I'm NOT suggesting we document fields used internally by the legal team in the spec. Those should only be documented in the license-list-XML repo.

If there are differences in the set of fields, simply copying the XML over (as the first comment on this issue) suggests, won't work.


Personally, I would really like to have more discussion before declaring "the XML files as a stable standard format" and I don't think it should be only in the legal team, as it involves XML technical stuff. I mean, we should be publishing an ontology, with clear explanations on how `<alt>` works, etc.

**Comment by goneall** [comment]
@zvr Good point - my last comment is definitely out of date.  If we make the license XML official, all the fields should be documented.

I'm still traveling for the next 2 weeks with limited connectivity, perhaps when were back we can schedule a joint legal/tech team call to discuss both making the license list XML format official and how/where/why we document the specific fields.

One other topic would be whether we include the license XML within the JSON and RDF license metadata.  I didn't think about that possibility until just now.  We currently include the template text (which we would re-characterize as "legacy template" format once we support license XML).

**Comment by swinslow** [comment]
@goneall @zvr I'm on-board with us setting up a joint legal/tech team call at some point once @goneall is back, to discuss these details about formalizing and making "official" the license list XML format. I'll mention this during the legal team call today so that others are aware.

**Comment by jlovejoy** [comment]
agree with @swinslow - this would be good to discuss on a joint call. it also relates to some questions that came up when I was updating the matching guidelines to reference the XML tags. there are more tags than are "relevant" to the matching guidelines that could use better documentation, so I think that's kind of part of this?

**Comment by swinslow** [comment]
No further discussion on this (to my recollection) during the 3.19 documentation release cycle. Moving to "Later Release".

Looks like next steps here should be:
* joint call with participants from Legal and Tech Team, to determine whether to proceed with this
* if so, may be necessary to improve documentation around the XML fields. We've added [semi-formal guidance documentation about this](https://github.com/spdx/license-list-XML/blob/main/DOCS/xml-fields.md) during 3.19, and the [XML schema file](https://github.com/spdx/license-list-XML/blob/main/schema/ListedLicense.xsd) is available, but to be discussed if more is needed.

**Comment by swinslow** [comment]
@zvr @goneall -- dusting off this old issue from 2022. It looks like part of the discussion here was about creating documentation to make the license XML format more "official" or formalized. Is there more here that you think we should do, beyond the schema file + existing guidance document? If so, happy to discuss, otherwise it may make sense for us to close this issue.

**Comment by goneall** [comment]
@swinslow  - this has already been implemented: https://github.com/spdx/license-list-data/tree/main/license-list-XML

I'll close this as resolved.
